### PR TITLE
[DUOS-446][risk=no] Remove unused DAA code

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
UI side of https://broadinstitute.atlassian.net/browse/DUOS-446
See also: https://github.com/DataBiosphere/consent/pull/753

## Changes
* Mostly code removal. We no longer upload or download a DAA for the user, that code has been removed from the UI for some time.
* Remove the download DAA link on researcher review page. This is technically no longer a valid function.
  * Have received Jonathan's 👍 on removing this feature

---- 

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] PR is labeled with a Jira ticket number and includes a link to the ticket
- [x] PR is labeled with a security risk modifier [no, low, medium, high] 
- [x] PR describes scope of changes

In all cases:

- [x] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [x] Get PO sign-off for all non-trivial UI or workflow changes
- [x] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/DatasetRegistration.js [hunk 1]
```diff
@@ -70,8 +70,6 @@ class DatasetRegistration extends Component {
         havePi: '',
         profileName: '',
         piName: '',
-        urlDAA: '',
```
Comment (JVThomas) [inline src/pages/DatasetRegistration.js:73]: DatasetRegistration has a state variable called ```file``` that is also tied to the DAA models (similar to DataAccessRequestRenewal). Should that also be removed?

**Comment by JVThomas** [comment]
One last comment: There are some css files that have classes associated with DAAs that should be removed if the model is no longer rendered on the front-end

File: src/pages/DatasetRegistration.js (hunk 1 above)
Comment (lindojong) [reply src/pages/DatasetRegistration.js:73]: I think it's safe to remove! A lot of the state in DatasetRegistration won't end up being used. I was going to remove it in my own PR if Greg doesn't do it here.

Comment (rushtong) [reply src/pages/DatasetRegistration.js:73]: Yes, that should go too, I missed that one

Comment (rushtong) [reply src/pages/DatasetRegistration.js:73]: Done!

**Comment by rushtong** [comment]
Found the unused styles. There are some other related ones that are still used by other file related features, so only removing the `daa` ones.

**Review by JVThomas — APPROVED** [review]

**Pull request merged**